open-company/open-company-web
# Fix more menu icons and position of more, activity and move views.

From: https://paper.dropbox.com/doc/Misc-UX--ANCZ6eHtJr7AJ~F8CVomKn~hAg-zmqEXqPjOZGPJJSxUfPxC

Item:
> Make ellipsis and share more visible on hover https://invis.io/VCO38CFA6M8

Abstract: https://share.goabstract.com/a823001b-f21a-4d51-90af-6d4827f3dd05

To test:
- go to AP or a board with some posts
- [ ] check more menu
- [ ] check activity popup
- [ ] check move popup
- go to fullscreen view (share and copy url)
- [ ] check more menu
- [ ] check activity popup
- [ ] check move popup
